### DISCUSSION TITLE
Flag to use to disable default style

### DISCUSSION THREAD

**Discussion by ylbeethoven** [charmbracelet/gum#238] [Q&A]
Hi,

I am using `gum style` to create banners for my scripts. Because I need many banners, I am using environment variables to set the same style.

However these env variable applies to ALL `gum style` command which means I need to manually pass in default values if I want to reset styling.

Does anyone know if there is a flag to reset the style?

Thank you.

**Comment by muesli** [comment]
I'd only set the env vars required for each `gum` call and then unset them again with `unset`, e.g. `unset GUM_STYLE`.

**Reply by ylbeethoven** [reply]
Thanks Christian, just want to get your view on the best practice for styling.

Previously I was using flags like `--margin`  `--foreground` with `gum style` command. 

> set the env vars required for each gum call

I am curious of why you set env vars for each gum call instead of using flags.

By the way, the workaround I am using is to write a shell script to handle different styles.

`gum_style "<content>" "<style>"`

Thank you.
